### PR TITLE
Add feature to start task

### LINKED ISSUE #320: Feature Request: Ability to Start Task Run via CLI
This issue is a feature request to be able to start a task run via `tkn` similar to how a pipeline can be started (i.e. `tkn pipeline start`). The idea would be completely that same as `tkn pipeline start`, but just to execute an individual task. Pipeline resources and a service account could be provided via `-r` and `-s`. 

# Expected Behavior

`tkn task start task-name -r pipelineresource1 -r pipelineresource2 -s serviceaccount` 

Create a task run by providing needed pipeline resources and service account by running a `tkn task start` command. The command will execute an individual task in a particular namespace.

### PR DESCRIPTION
**Pull request by piyush-garg**
# Changes

This will add the feature of starting
a task or creating a taskrun from cli

Command will be like `tkn task start ..`

Add -i flag for input resources
Add -o flag for output resources
Add -p flag for param
Add -l flag for labels
Add --last flag for starting on basis of last run
Add -s flag for serviceaccount

Add docs and tests

Refactoring to remove duplicate code

Fix #320

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Regenerate the manpages and docs with `make docs` and `make man` if needed.
- [x] Run the code checkers with `make check`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
Feature to start task from cli
```

### CODE REVIEW

**Comment by piyush-garg** [comment]
/test pull-tekton-cli-unit-tests
--

File: pkg/cmd/pipeline/start.go [hunk 1]
```diff
@@ -26,6 +26,8 @@ import (
 	"github.com/tektoncd/cli/pkg/cli"
 	"github.com/tektoncd/cli/pkg/cmd/pipelinerun"
 	"github.com/tektoncd/cli/pkg/flags"
+	"github.com/tektoncd/cli/pkg/helper/labels"
```
Comment (vdemeester) [inline pkg/cmd/pipeline/start.go]: nit: I am not a huge fine of the `helper` package but it's already there, so can be changed in a follow-up anyway :wink:

Comment (piyush-garg) [reply pkg/cmd/pipeline/start.go]: This is required for both, `start pipeline` and `start task` so I put this in helper

**Pull request merged**